cloud/amazon/ec2_elb_lb.py

```diff
@@ -708,6 +719,12 @@ def _set_connection_draining_timeout(self):
             attributes.connection_draining.enabled = False
             self.elb_conn.modify_lb_attribute(self.name, 'ConnectionDraining', attributes.connection_draining)
 
+    def _set_idle_timeout(self):
+        attributes = self.elb.get_attributes()
+        if self.idle_timeout is not None and self.idle_timeout != attributes.connecting_settings.idle_timeout:
```

Comment:
@digital-abyss This is my bad. I always think not None looks bad until I see the workarounds to check if a variable is defined. I am not sure except NameError or if None else are really cleaner. I should take this up with the Python gods. If Ansible can have is defined, why can't Python have is_defined()?

So don't stress over this. You can leave it as is.